protos/flyteidl/admin/task.proto

```diff
@@ -51,6 +52,9 @@ message TaskList {
 message TaskSpec {
     // Template of the task that encapsulates all the metadata of the task.
     core.TaskTemplate template = 1;
+
+    // DescriptionEntity encapsulates all the detailed documentation for the task.
+    DescriptionEntity description_entity = 2;
```

Comment:
I'm a bit confused about this. Perhaps because I'm not sure how this is planned to work out.
There are APIs to explicitly create/retrieve a DescriptionEntity but then the DescriptionEntity is already included in the Admin objects... so which is it? why do we have two paths to do the same thing...